specification/e-referrals-service-api.yaml

```diff
@@ -537,9 +537,133 @@ paths:
+
+        | File type          | File extension(s)           |
+        | ------------------ | --------------------------- |
+        | Plain Text         | `.txt`                      |
+        | HTML Text          | `.html,` `.htm`             |
+        | PDF                | `.pdf`                      |
+        | XML Text           | `.xml`                      |
+        | RTF Text           | `.rtf`, `.rtx`              |
+        | Basic Audio        | `.pdf`                      |
+        | MPEG audio layer 3 | `.mp3`                      |
+        | PNG Image          | `.png`                      | 
+        | GIF Image          | `.gif`                      |
+        | JPEG Image         | `.jpg,` `.jpe`, `.jpeg`     |
+        | TIFF Image         | `.tif`, `.tiff`             |
+        | MPEG Video         | `.mpg`, `.mpeg`, `.mpe`     |
+        | MSWORD             | `.doc`, `.docx`             |
+
+        Currently, the maximum file size supported by e-RS is 5MB.
+
+        ## Use case
+
+        As an authenticated e-RS user
+
+        I need to upload a referral letter or other clinical information file to e-RS
+
+        So that I can associate the file with an existing referral.
+
+        ## Important notes:
+
+        This endpoint **only** performs the file upload. The uploaded file is **not** associated with a referral until the file has been associated with a referral, for example using [[HYPERLINK_A012]].
+        
+        The resulting URL within e-RS for the uploaded file is returned in the 'Location' header of the response.
+
+        ## Related endpoints
+
+        You can associate the uploaded file with a specific referral using [[HYPERLINK_A012]].
+
+      summary: A020 - Upload file to document store
+      operationId: a020-Upload-file-to-document-store
+      parameters:
+        - $ref: 'components/schemas/headers/request/BearerAuthorization.yaml'
+        - $ref: 'components/schemas/headers/request/OdsCode.yaml'
+        - $ref: 'components/schemas/headers/request/BusinessFunction.yaml'
+        - $ref: 'components/schemas/headers/request/CorrelationID.yaml'
+        - $ref: 'components/schemas/headers/request/ErsXapiMetaFilename.yaml'
+        - $ref: 'components/schemas/headers/request/ErsXapiMetaIntendedUbrn.yaml'
+      requestBody:
+        required: true
+        content:
+          text/plain:
+            schema:
+              $ref: '#/components/schemas/StringBinarySchema'
+            example: 'This is some example text.'
+          text/html:
+            schema:
+              $ref: '#/components/schemas/StringBinarySchema'
+          application/pdf:
+            schema:
+              $ref: '#/components/schemas/StringBinarySchema'
+          text/xml:
+            schema:
+              $ref: '#/components/schemas/StringBinarySchema'
+          text/rtf:
+            schema:
+              $ref: '#/components/schemas/StringBinarySchema'
```

Comment:
In the case of RTF, are there any restrictions on the character encoding that must be used?

Reply:
@adamoldfield-nhs  do you happen to know the answer to this question ?